docs/devel/developer-guides/vagrant.md

```diff
@@ -184,7 +184,7 @@ NAME   IMAGE(S)   HOST   LABELS   STATUS
 $ cluster/kubectl.sh get services
 NAME   LABELS   SELECTOR   IP   PORT
 
-$ cluster/kubectl.sh get replicationControllers
```

Comment:
We should really remove all stuff that's not specific to Vagrant here. It's going to go stale in all kinds of ways. For example, "get minions" (above) is now "get nodes".